apps/analysis_framework/enums.py

```diff
@@ -10,8 +10,8 @@
 )
 
 
-AnalysisFrameworkRoleTypeEnum = convert_enum_to_graphene_enum(
-    AnalysisFrameworkRole.Type, name='AnalysisFrameworkRoleTypeEnum')
+AnalysisFrameworkRolesTypeEnum = convert_enum_to_graphene_enum(
+    AnalysisFrameworkRole.Type, name='AnalysisFrameworkRolesTypeEnum')
```

Comment:
Type name shouldn't be plural.
```suggestion
AnalysisFrameworkRoleTypeEnum = convert_enum_to_graphene_enum(
    AnalysisFrameworkRole.Type, name='AnalysisFrameworkRoleTypeEnum')
```